examples/templates/devcontainer-docker/main.tf

```diff
@@ -89,14 +93,13 @@ EOF
 
 variable "cache_repo" {
   default     = ""
-  description = "Use a container registry as a cache to speed up builds."
-  sensitive   = true
+  description = "(Optional) Use a container registry as a cache to speed up builds."
```

Comment:
review: this shouldn't contain any credentials and it's kind of annoying to have this sensitive